### DISCUSSION TITLE
error NU3018 Linux

### DISCUSSION THREAD

**Discussion by alvahtin** [AvaloniaUI/Avalonia#14482] [Q&A]
I created new project

`dotnet new avalonia.mvvm -o GetStartedApp`

I tried to build the project

`dotnet run`

But I got a lot of errors. All of them error NU3018.

What certificates are needed? What is missing?

![image](https://github.com/AvaloniaUI/Avalonia/assets/50268307/95f03b69-ed0c-4045-9fab-72659fb162ae)

**Comment by thevortexcloud** [comment]
This looks like a nuget problem, not an Avalonia one. Please make sure your system is up to date as you may be missing a root certificate. Otherwise I believe it's somehow possible on newer versions of Nuget to tell it to ignore certificate validation, but it's highly not recommended.

**Reply by thevortexcloud** [reply]
Oh also a lot of enterprise networks will actually intercept HTTPS traffic and therefore require you to install their certs to do pretty much anything (which somewhat defeats the point of end to end encryption). Typically they only bother setting Windows machines to have the cert. So it's probably worth checking if that's the case.

**Comment by alvahtin** [comment]
I don't know how to update certificates. I added the certificate manually from https://github.com/dotnet/sdk/blob/main/src/Layout/redist/trustedroots and it worked.

Thank to everyone who responded.
